url.bs

```diff
@@ -85,6 +85,170 @@ valid input. User agents, especially conformance checkers, are encouraged to rep
  unclear to other developers.
 </div>
 
+<table>
+ <thead>
+  <tr>
+   <th>Error Code
+   <th>Error Message
+ <tbody>
+  <tr>
+   <td><dfn>unexpected-c0-control-or-space</dfn>
+   <td>This error occurs when the input to the URL parser contains leading or trailing
+   <a>C0 control or space</a> code points. The URL parser subsequently strips any matching code
+   points.
+  <tr>
+   <td><dfn>unexpected-ascii-tab-or-newline</dfn>
+   <td>This error occurs when the input to the URL parser contains <a>ASCII tab or newline</a> code
+   points. The URL parser subsequently strips any matching code points.
+  <tr>
+   <td><dfn>invalid-scheme-start-code-point</dfn>
+   <td>This error occurs when the first code point of a URL's <a>scheme</a> is not an
+   <a>ASCII alpha</a> code point.
+  <tr>
+   <td><dfn>file-scheme-missing-following-solidus</dfn>
+   <td>This error occurs when the URL parser encounters a URL with a "<code>file</code>"
+   <a>scheme</a>, but it is not followed by two "<code>//</code>" U+002F (/) code points.
+  <tr>
+   <td><dfn>invalid-scheme</dfn>
+   <td>This error occurs when the <a>scheme</a> contains a code point that is not an
+   <a>ASCII alphanumeric</a>, U+002B (+), U+002D (-), or U+002E (.) code point, or is not followed
+   by a U+00A3 (:) code point.
+  <tr>
+   <td><dfn>missing-scheme-non-relative-url</dfn>
+   <td>This error occurs when no <a>scheme</a> is present and either no <a>base URL</a>
+   was provided or the <a>base URL</a> cannot be used as a <a>base URL</a>.
+  <tr>
+   <td><dfn>relative-url-missing-beginning-solidus</dfn>
+   <td>This error occurs when a <a>relative-URL String</a> is given that does not begin with a
+   U+002F (/) code point.
+  <tr>
+   <td><dfn>wrong-solidus-special-scheme</dfn>
+   <td>This error occurs when a URL with a <a>special scheme</a> uses the U+005C (\) reverse
+   solidus code point instead of the U+002F (/) code point.
+  <tr>
+   <td><dfn>missing-solidus-before-authority</dfn>
+   <td>This error occurs if the URL <a>includes credentials</a> and they are not preceeded by
+   two "<code>//</code>" U+002F (/) code points.
+  <tr>
+   <td><dfn>unexpected-@-code-point</dfn>
```

Comment:
I think this should be commercial-at if we're not using / for solidus.

Reply:
Makes sense.